.github/workflows/integration-test.yml

```diff
@@ -5,6 +5,10 @@ on:
     branches: [ master ]
   workflow_dispatch:
 
+concurrency:
+  group: integration-test-${{ github.ref_name }}
+  cancel-in-progress: true
```

Comment:
Why is this required?

Reply:
yeah, since the workflows are segregated per sample app and makes it little easy to troubleshoot. instead of running multiple runs at the same time given PRs are merged at the same time. Thought we could use this change